platform/security/platform-security-core-api/src/main/java/ddf/security/SubjectUtils.java

```diff
@@ -106,4 +135,29 @@ public static String getName(org.apache.shiro.subject.Subject subject, String de
         LOGGER.debug("Sending back name {}.", name);
         return name;
     }
+
+    public static String getCommonName(X500Principal principal) {
```

Comment:
We should probably refactor this method to use X500Name from bouncy castle since we're hitting it anyways, this is how you would do it:

    X500Name x500Name = new X500Name(principal.getName())
    String cn = x500Name.getRDNs(BCStyle.CN)[0];